composer.json

```diff
@@ -1,11 +1,15 @@
 {
-    "name": "graylog2/gelf-php",
-    "description": "A php implementation to send log-messages to a GELF compatible backend like Graylog2.",
+    "name": "proximedia/gelf-php",
```

Comment:
This needs to be on a different branch then. Github PRs are always between the latest commits of the specified branches.